addons/voice/pom.xml

```diff
@@ -17,7 +17,8 @@
   <name>openHAB Voice Add-Ons</name>
 
   <modules>
-    <!--<module>org.openhab.voice.kaldi</module>-->
+    <module>org.openhab.voice.gtts</module>
```

Comment:
Maybe it is still work in progress, but the project dir still needs to be renamed.